### ISSUE TITLE
add parser for bracketed logs in v1

### ISSUE THREAD

**Issue by rgerhards** [rsyslog/liblognorm#76]
Some logs are bracketed via "[...][...]". We should support this even in liblognorm v1, albeit with a bit limited features. For v2, we should do a full implementation (due to better config options), but that's for another issue tracker.

**Comment by rgerhards** [comment]
User request is not urgent, so we may want to skip this until v2.

**Comment by davidelang** [comment]
v2 features make this much easier to deal with. unless user defined types are significantly slower than built-in types, I think we want to limit the explosion of built-in types and instead start including a library of common user defined types that can be included/tweaked

**Comment by rgerhards** [comment]
Yes, that's the idea of v2, but I couldn't express this explicitly or
implement anything before I left for vacation.

Sent from phone, thus brief.
Am 22.08.2015 06:39 schrieb "davidelang" <notifications@github.com>:
